File: ianisnotgay.py
import math
def calculatr(one, two, oper):
  if oper == '*':
    result = one * two
  elif oper == '^':
    result = pow(one, two)
  elif oper == '%':
    result = one % two
  
  return result
#MAIN FUNCTION DONT MESS UP PLZ IAN
operator = input("operator")
first = input("first number")
second = input("second number")
first = int(first)
second = int(second)
myresult = calculatr(first, second, operator)
print(myresult)

File: ianisnotgay.cpp
#include <iostream>
using namespace std;
int calculatr(int one, int two, char oper) {
int result;
  if (oper == '*') {
    result = one * two;
  }
  if (oper == '^') {
    result = one ^ two;
  }
  if (oper == '%') {
    result = one % two;
  }
  return result;
}
int main() {
  char oprator;
  int first, second;
  cout << "operator\n";
  cin >> oprator;
  cout << "first number\n";
  cin >> first;
  cout << "second number\n";
  cin >> second;
  cout << "answer: " << calculatr(first, second, oprator) << endl;
  return 0;
}
